src/handlers/gestureHandlerCommon.ts

```diff
@@ -180,3 +181,17 @@ export function findNodeHandle(
   if (Platform.OS === 'web') return node;
   return findNodeHandleRN(node);
 }
+
+let scheduledHandlersQueueFlush:
+  | ReturnType<typeof requestAnimationFrame>
+  | undefined = undefined;
+
+export function scheduleHandlersQueueFlush() {
```

Comment:
Once `flushHandlersQueue` method is renamed, don't forget to propagate this change here as well.

I'm also thinking if the pattern of the name of this function should be `scheduleFlushSomethingQueue` or `scheduleSomethingQueueFlush`.

Personally I like the first option better since it will also appear in the search results for `flushSomethingQueue` (at least for case-insensitive search).

Reply:
Done in https://github.com/software-mansion/react-native-gesture-handler/pull/1979/commits/da1592520ebf7f94e9a060a68b5bbe83efc6d2a1.